pkg/otlp/metrics/sketches_test.go

```diff
@@ -187,21 +195,27 @@ func TestHistogramSketches(t *testing.T) {
 	}
 }
 
-func TestExactSumCount(t *testing.T) {
+func TestExactSummaryStatistics(t *testing.T) {
```

Comment:
Test name is a bit confusing as this is testing histogram rather than summary. Consider sth. like TestExactHistogramStats

Reply:
Good point, I was using the term in [this sense](https://en.wikipedia.org/wiki/Summary_statistics) but it's definitely confusing. Renamed with your suggestion